templ_enum.cif

```diff
@@ -856,6 +858,7 @@ save_units_code
 
    'counts_per_photon' "measure of gain used in array detectors"
    'counts'            "counts from a detector"
+   'counts_per_second' "counts from a detector per second of count time"
```

Comment:
The `counts_per_second` units are only required if the a certain PR in the `CIF_POW` dictionary is approved (https://github.com/COMCIFS/Powder_Dictionary/pull/156). Let's wait for that PR to be approved (on a conceptual level) before merging this PR.

Reply:
Ok, so the PR in question has now been merged.